package.json

```diff
@@ -32,6 +32,7 @@
     "@types/papaparse": "4.1.33",
     "@types/webcrypto": "^0.0.28",
     "angular2-template-loader": "^0.6.2",
+    "braintree-web-drop-in": "^1.13.0",
```

Comment:
Shouldn't this be a regular dependency and not a dev?

Reply:
I guess? The JS gets copied over during build time. I'm happy to update it if you want

Reply:
The `qrious` library you are modeling this after is in dependencies. please add it there too.